packages/noir-ethereum-api/src/noir/oracles/server.ts

```diff
@@ -1,17 +1,8 @@
-import { JSONRPCRequest, JSONRPCServer, TypedJSONRPCServer } from 'json-rpc-2.0';
-import Fastify from 'fastify';
-import { getHeaderHandler, getAccountHandler, JSONRPCServerMethods, ServerParams } from './server/handlers.js';
-import { createDefaultClient } from '../../ethereum/client.js';
+import { buildFastifyApp } from './server/app.js';
 
 const PORT = 5555;
-const client = createDefaultClient();
 
-const jsonRPCServer: TypedJSONRPCServer<JSONRPCServerMethods, ServerParams> = new JSONRPCServer();
-const serverParams = { client };
-jsonRPCServer.addMethod('get_header', getHeaderHandler);
-jsonRPCServer.addMethod('get_account', getAccountHandler);
-
-const app = Fastify({
+const app = buildFastifyApp({
```

Comment:
I would propose rename `buildOracleServer`, as we can expect to have more than one server, i.e. `buildProverServer` would be the other one

Reply:
Agree. Changed